client/src/ui/components/Spinner.tsx

```diff
@@ -1,38 +1,33 @@
 import React from 'react';
-import cx from 'classnames';
+import styled, { keyframes } from 'styled-components';
 
 interface SpinnerProps {
-  className?: string;
   size: 'small' | 'large';
 }
 
-export const Spinner = ({ className, size }: SpinnerProps) => (
-  <React.Fragment>
-    <style>{`.loader {
-  border-top-color: #1a202c;
-  -webkit-animation: spinner 1.5s linear infinite;
-  animation: spinner 1.5s linear infinite;
-}
+const rotate360 = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
 
-    @-webkit-keyframes spinner {
-  0% { -webkit-transform: rotate(0deg); }
-  100% { -webkit-transform: rotate(360deg); }
-}
-
-@keyframes spinner {
-  0% { transform: rotate(0deg); }
-  100% { transform: rotate(360deg); }
-}`}</style>
+// ${({ theme }) => theme.background};
+const StyledSpinner = styled.div`
+  animation: ${rotate360} 1.5s linear infinite;
+  height: ${(props: SpinnerProps) =>
+    props.size === 'small' ? '3rem' : '6rem'};
+  width: ${(props: SpinnerProps) => (props.size === 'small' ? '3rem' : '6rem')};
+  border-radius: 9999px;
+  border-width: ${(props: SpinnerProps) =>
+    props.size === 'small' ? '0.3rem' : '0.5rem'};
+  border-top-width: ${(props: SpinnerProps) =>
+    props.size === 'small' ? '0.3rem' : '0.5rem'};
+  border-top-color: #1a202c;
+  color: white;
+  transition-timing-function: linear;
+`;
 
-    <div
-      className={cx(
-        'loader ease-linear rounded-full border-gray-200 ',
-        {
-          'border-8 border-t-8 h-24 w-24': size === 'large',
-          'border-4 border-t-4 h-8 w-8': size === 'small',
-        },
-        className
-      )}
-    ></div>
-  </React.Fragment>
-);
+export const Spinner = (props: SpinnerProps) => <StyledSpinner {...props} />;
```

Comment:
@Akirtovskis to mix both we could do something like this:
```
export const Spinner = (props: SpinnerProps) => <StyledSpinner className={cx(
        'loader ease-linear rounded-full border-gray-200 ',
        {
          'border-8 border-t-8 h-24 w-24': size === 'large',
          'border-4 border-t-4 h-8 w-8': size === 'small',
        },
        className
      )} {...props} />;
```

What do you think?